packages/dappmanager/src/modules/https-portal/utils.ts

```diff
@@ -26,6 +26,16 @@ export function removeNetworkAliasCompose(
   compose.write();
 }
 
+export function composeVersionMigration(container: PackageContainer): void {
```

Comment:
This function on it's own is confusing, who-ever uses the `name` feature should ensure that the docker-compose version is at least 3.5